### ISSUE TITLE
Discover Searching Opens Results in App

### ISSUE THREAD

**Issue by delianides** [NewSpring/Apollos#658]
Right now discover search results open their links through the in app browser instead of the app itself.

**Comment by jonhorton** [comment]
Just spotted this too—opening results (and other links) within the app will be a game changer for discovering other content!

**Comment by johnthepink** [comment]
Totally. We also need to update our redirecting so content from Perry's site doesn't try to go to newspring.cc https://github.com/NewSpring/Apollos/issues/498

**Comment by johnthepink** [comment]
Actually, the redirects work fine in the app because of the in app browser

**Comment by jonhorton** [comment]
@johnthepink was this resolved? should discover results go to in app pages if it's an articles/devo/story/etc?

**Comment by delianides** [comment]
@Jonhorton no. Discover still uses the in app browser.

**Comment by johnthepink** [comment]
@jonhorton @delianides correct. We need to intelligently parse external links and internalize them if possible. Not totally trivial since the app and the website have different url schems
